src/test/app/claims/claimStoreClient.ts

```diff
@@ -15,6 +15,10 @@ import { Claim } from 'claims/models/claim'
 import { ClaimData } from 'claims/models/claimData'
 import { InterestType as ClaimInterestType } from 'claims/models/interestType'
 import * as moment from 'moment'
+import { sampleOrdersDraftObj } from '../../http-mocks/draft-store'
+import { RequestedBy, ReviewOrder } from 'claims/models/reviewOrder'
+import { OrdersDraft } from 'orders/draft/ordersDraft'
+import { resolveSaveOrder, sampleClaimIssueObj } from '../../http-mocks/claim-store'
```

Comment:
Refactor away the relative imports please

Reply:
done